Gruntfile.js

```diff
@@ -362,7 +362,21 @@ module.exports = function (grunt) {
         browsers: [ 'PhantomJS' ],
         singleRun: true
       }
+    },
+
+    // Server Tests
+    mochaTest: {
+      test: {
+        options: {
+          reporter: 'spec',
+          //captureFile: 'results.txt', // Optionally capture the reporter output to a file
+          quiet: false, // Optionally suppress output to standard out (defaults to false)
+          clearRequireCache: false // Optionally clear the require cache before running tests (defaults to false)
+        },
+        src: ['common/models/test/**/*.js']
```

Comment:
This should be `server/test/*.js`, the usual convention is to pick up only top-level JS files.

Tests in `common` folder would be run by `grunt test:common`, but don't worry about that now.